src/ApiResourceExampleDocument.js

```diff
@@ -663,11 +690,13 @@ export class ApiResourceExampleDocument extends AmfHelperMixin(LitElement) {
     let parts = 'content-action-button, code-content-action-button, content-action-button-disabled, ';
     parts += 'code-content-action-button-disabled content-action-button-active, ';
     parts += 'code-content-action-button-active, code-wrapper, example-code-wrapper, markdown-html';
+    const classMaxHeight = this.maxHeight ? 'customMaxHeight' : '';
+
     return examples.map((item) => html`
     <div class="item-container">
       ${this._titleTemplate(item)}
       ${this._descriptionTemplate(item)}
-      <div class="renderer">
+      <div class="renderer ${classMaxHeight}">
```

Comment:
And this isn't necessary. If you use the CSS variable in the CSS file, you don't need to configure anything via JavaScript. Just having the `max-height: var(--api-resource-example-document-max-height, initial);` is enough for this to work without touching the JavaScript, and you don't need to add any additional properties to this component

Reply:
Done, deleted